### PR TITLE
test bus error handling

### PR DESCRIPTION
**Pull request by sduthil**
Why:

* If the bus thread does not correctly catch errors, it will stop and
never process any more events

### CODE REVIEW

File: wazo_webhookd/bus.py [hunk 1]
```diff
@@ -155,3 +158,13 @@ def _create_bindings(self, event_names, user_uuid, wazo_uuid):
             result.append(binding)
 
         return result
+
+    @staticmethod
+    def no_error_callback(callback, body, message):
```
Comment (fblackburn1) [inline wazo_webhookd/bus.py:163]: Not a standard, but using same naming than other services would simplify comprehension (agentd, calld, chatd, confd, phoned)
```suggestion
    def _on_bus_message(callback, body, message):
```

File: tox.ini [hunk 2]
```diff
@@ -43,7 +43,9 @@ commands =
 
 [testenv:integration]
 usedevelop = true
-deps = -rintegration_tests/test-requirements-for-tox.txt
+deps = 
+  -rintegration_tests/test-requirements-for-tox.txt
+  -eintegration_tests/plugins/sentinel_client
```
Comment (fblackburn1) [inline tox.ini:48]:
```suggestion
deps =
    -rintegration_tests/test-requirements-for-tox.txt
    -eintegration_tests/plugins/sentinel_client
```

**Review by fblackburn1 — CHANGES_REQUESTED** [review]
Nice tests :+1:

File: wazo_webhookd/bus.py (hunk 1 above)
Comment (fblackburn1) [reply wazo_webhookd/bus.py:163]: Oh I see the origin (dird) ... maybe it's time to make standard :trollface:

Comment (sduthil) [reply wazo_webhookd/bus.py]: Standard is good :) Thanks for noticing!

File: tox.ini (hunk 2 above)
Comment (sduthil) [reply tox.ini]: Fixed.

**Review by fblackburn1 — APPROVED** [review]

**Comment by fblackburn1** [comment]
regate

**Pull request merged**